tests/components/bond/test_cover.py

```diff
@@ -89,3 +111,57 @@ async def test_stop_cover(hass):
         )
         await hass.async_block_till_done()
         mock_hold.assert_called_once()
+
+
+async def test_update_reports_open_cover(hass: core.HomeAssistant):
+    """Tests that update command sets correct state when Bond API reports cover is open."""
+
+    await async_setup_component(hass, HA_DOMAIN, {})
```

Comment:
Why set up the home assistant domain?

Reply:
If I don't setup HA domain, then I cannot call SERVICE_UPDATE_ENTITY service to cause polling on the entity - it will report that "could not find the service". Is there a different suggested way to cause polling in the test?

Reply:
We can move time forward by firing a time changed event:

https://github.com/home-assistant/core/blob/08fa701854d420c36aa539ab06166780cb7274db/tests/common.py#L288

Reply:
Got it, changed.